Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -777,12 +860,83 @@ - (nullable UIView *)createTapForMoreLabelIfNecessary
     tapForMoreLabel.text = NSLocalizedString(@"CONVERSATION_VIEW_OVERSIZE_TEXT_TAP_FOR_MORE",
         @"Indicator on truncated text messages that they can be tapped to see the entire text message.");
     tapForMoreLabel.font = [self tapForMoreFont];
-    tapForMoreLabel.textColor = [self.textColor colorWithAlphaComponent:0.85];
+    tapForMoreLabel.textColor = [self.bodyTextColor colorWithAlphaComponent:0.85];
     tapForMoreLabel.textAlignment = [tapForMoreLabel textAlignmentUnnatural];
 
     return tapForMoreLabel;
 }
 
+- (UIView *)createQuotedMessageView
+{
+    OWSAssert(self.isQuotedReply);
+
+    UIView *quotedMessageView = [UIView containerView];
+    quotedMessageView.userInteractionEnabled = NO;
+    quotedMessageView.clipsToBounds = YES;
+    // TODO:
+    quotedMessageView.layer.cornerRadius = 3.f;
+    quotedMessageView.backgroundColor = [UIColor colorWithRGBHex:0xe2f7fa];
+
+    UIView *quoteStripView = [UIView containerView];
+    quoteStripView.backgroundColor = (self.isIncoming ? [UIColor whiteColor] : [UIColor colorWithRGBHex:0x007884]);
```

Comment:
nit: pull 0x007884 into a UIColor constant, and below, or maybe you're waiting for the final color palette before pulling these out.

Reply:
Exactly, these colors are temporary.